lte/gateway/python/magma/pipelined/app/uplink_bridge.py

```diff
@@ -230,6 +238,18 @@ def _set_sgi_interface_ingress_flows(self):
             actions = "output:LOCAL"
             self._install_flow(flows.MEDIUM_PRIORITY + 1, match, actions)
 
+        if_addrs = netifaces.ifaddresses(self.config.uplink_bridge).get(netifaces.AF_INET6, [])
+        for addr in if_addrs:
+            addr = addr['addr']
+            if '%' in addr:
```

Comment:
if % not in addr what happens here? Lets either check the return(if not % in addr: exit) or remove this if ?

Reply:
its optional, so it does not matter.